docs/how-to/topics/file-management.rst

```diff
@@ -0,0 +1,206 @@
+                on_press=self.save_button_pressed,
+                margin=20,
+            )
+
+            main_box = toga.Box(
+                direction=COLUMN,
+                children=[self.text_input, self.config_directory, load_button, save_button],
+            )
+            self.main_window = toga.MainWindow(content=main_box)
+            self.main_window.show()
+
+        def load_button_pressed(self, button, **kwargs):
+            path = Path("/Users/brutus/config-file-creator/initial_config.toml")
+            self.text_input.value = path.read_text(encoding="utf-8")
+
+        def save_button_pressed(self, button, **kwargs):
+            self.config_directory.value = "Save not implemented."
+
+
+    def main():
+        return ConfigFileCreator()
+
+
+This creates a multi-line text input with a button that loads the contents of an ``initial_config.toml`` file into the text input. The file content is read from the file using ``pathlib.Path().read_text()`` with a hard-coded *absolute* file path.
+
+Now, if we run ``briefcase dev``, the app will fail to start, resulting in a ``FileNotFoundError``. This version worked great on Brutus' computer. However, the moment we try to start the app on a different machine, the path is no longer valid, and it will fail to run when it can't find the file.
+
+You might be thinking, the issue is that the file is not located in the application. Let's examine this as a possibility. Here is a the basic structure of a Briefcase project:
+
+.. code-block:: console
+
+    configfilecreator/
+    └── src/
+        └── configfilecreator/
+            ├── app.py
+            └── resources/
+
+One possible option is in the top level ``configfilecreator/`` directory, as that's the location from which we actually run the app. This may have been what Brutus was intending with the path they specified. While we could point our code to this location as an absolute path, we will still run into the problem when running the app from anywhere else but our own computer.
+
+A second possible option might be to put the file in ``configfilecreator/src/configfilecreator`` because that's where the ``app.py`` file is. After all, Python bases file access on the directory from which the program is being run. While the second option does ensure Briefcase packages the file with the app, it still doesn't guarantee a consistent path.
+
+To avoid the possibility of either of the above happening accidentally, Briefcase sets the current working directory to elsewhere so we aren't caught by this issue.
+
+Let's take a look at an example of packaging the file content with the app, and using ``__file__`` to locate it.
+
+Create an ``initial_config.toml`` file containing the following content, and place it in the ``resources/`` directory within the Briefcase project:
+
+.. code-block:: toml
+
+    # Update the following to match your configuration
+    PROJECT_NAME = "Name"
+    PROJECT_VERSION = "v0.0.0"
+    AUTHOR_NAME = "Your Name"
```

Comment:
If we're going to add another "relative that doesn't work" example, we might need this content further up the page.